File: Bulky.Models/Company.cs
using System.ComponentModel.DataAnnotations;

namespace BookWorld.Models
{
    public class Company
    {
        [Key]
        public int Id { get; set; }
        [Required]
		[Display(Name = "Назва")]
		public string Name { get; set; }
        [Display(Name = "Адреса")]
        public string? StreetAdress { get; set; }
		[Display(Name = "Місто")]
		public string? City { get; set; }
		[Display(Name = "Країна")]
		public string? State { get; set; }
        [Display(Name = "Поштовий індекс")]
        public string? PostalCode { get; set; }
        [Display(Name = "Номер телефону")]
        public string? PhoneNumber { get; set; }
    }
}

File: BulkyWeb/wwwroot/js/company.js
var dataTable;

$(document).ready(function () {
    loadDataTable();
});

function loadDataTable() {
    dataTable = $('#tblData').DataTable({
        "ajax": { url: '/admin/company/getall' },
        "columns": [
            { data: 'name', "width": "15%" },
            { data: 'streetAdress', "width": "15%" },
            { data: 'city', "width": "15%" },
            { data: 'state', "width": "15%" },
            { data: 'phoneNumber', "width": "10%" },
            {
                data: 'id',
                "render": function (data) {
                    return `<div class="w-60 btn-group" role="group">
                    <a href="/admin/company/upsert?id=${data}" class="btn btn-primary mx-2"> <i class="bi bi-pencil"></i> Редагувати</a>
                    <a onClick=Delete('/admin/company/delete/${data}') class="btn btn-primary"> <i class="bi bi-trash3""></i> Видалити</a>
                    <div>
                    `
                },
                "width": "21%"
            }
        ]
    });
}

function Delete(url) {
    Swal.fire({
        title: "Ви впевнені?",
        text: "Ви не зможене повернути видалений об'єкт!",
        icon: "warning",
        showCancelButton: true,
        confirmButtonColor: "#3085d6",
        cancelButtonColor: "#d33",
        confirmButtonText: "Так, видалити!",
        cancelButtonText: "Відмінити"
    }).then((result) => {
        if (result.isConfirmed) {
            $.ajax({
                url: url,
                type: 'DELETE',
                success: function (data) {
                    dataTable.ajax.reload();
                    toastr.success(data.message);
                }
            })
        }
    });
}